src/llmq/utils.cpp

```diff
@@ -1089,7 +1089,7 @@ void InitQuorumsCache(CacheType& cache)
 {
     for (const auto& llmq : Params().GetConsensus().llmqs) {
         cache.emplace(std::piecewise_construct, std::forward_as_tuple(llmq.type),
-                      std::forward_as_tuple(llmq.keepOldConnections));
+                      std::forward_as_tuple(llmq.keepOldKeys));
```

Comment:
I understand the change, but do you really think we should cache 2 months of Platform quorum's vvec + sk share for the eventual case of Platform restart?
In such case, we could still do the slow read, no need to cache imo.

```suggestion
                      std::forward_as_tuple(llmq. keepOldConnections));
```

Reply:
e81de93ee23e2505fe64dc8945ab4720d7eaeb81 should limit that to only 2 maps we really need to scan quorums